internal/service/objectstorage/objectstorage_object.go

```diff
@@ -303,7 +305,10 @@ func (o *objectResource) Update(ctx context.Context, req resource.UpdateRequest,
 }
 
 func (o *objectResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
-	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
+	bucketName, key := ObjectIDParser(req.ID)
```

Comment:
ID 입력을 잘 못한 경우 에러가 출력되면 좋겠네요

	idParts := strings.Split(req.ID, "/")

	if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" {
		resp.Diagnostics.AddError(
			"Unexpected Import Identifier",
			fmt.Sprintf("Expected import identifier with format: bucket-name/key Got: %q", req.ID),
		)
		return
	}

Reply:
요 경우에, object id의 경우 여러 개의 "/"를 포함하고 있어 "/"를 기준으로 split을 하면 len > 2 가 되어 에러를 반환합니다.

따라서 기존 ObjectIDParser를 사용하되(함수 내 다중 "/"에 대한 처리 포함), bucketName과 key가 빈 문자열인 경우에 대한 에러처리를 포함하였으니 참고 부탁드립니다! (다른 곳도 동일하게 적용)
```suggestion
	bucketName, key := ObjectIDParser(req.ID)

	if bucketName == "" || key == "" {
		resp.Diagnostics.AddError(
			"Unexpected Import Identifier",
			fmt.Sprintf("Expected import identifier with format: bucket-name/key Got: %q", req.ID),
		)
		return
	}
```

Reply:
이런 경우라면 "/" 대신에 다른 문자 ":" 를 구분자로 사용하여 bucket-name 과 key 를 구분하는게 더 좋을 것 같네요.

Reply:
당시 object id를 설정함에 있어 https://kr.object.ncloudstorage.com/tfstate-backend/media/test-test.jpg 같이 실제 접근을 위한 링크를 표현하는 방식과 동일하게 가져가고자 "/"를 사용한 바 있었습니다. 

개인적으로는 추후 사용에 용이하도록 "/" 를 적용해도 괜찮다고 생각하고 있는데, 해당 사항에 대해서도 한 번 의견 여쭙습니다 :)

아마 변경하게 된다면 예시로 말씀해주신 NRN의 기본 구분자인 ":"를 사용할 듯 합니다.

Reply:
@youngmn 제가 실수로 resolved로 두어서 노출되지 않은 듯 합니다 ㅜㅜ 리마인드 드립니다!!

Reply:
네 이건 유지하는게 좋겠네요